### DISCUSSION TITLE
Is this project under active development?

### DISCUSSION THREAD

**Discussion by trsh** [Toni-SM/skrl#143] [Q&A] [answered]
I want to use skrl for one of my upcoming projects, but the slower development activity since Aug 16, 2023 (v1.0.0) raises some doubts.  Would be nice, if devs can explain the current situation.

**Comment by Toni-SM** [comment, marked as answer]
Hi @trsh

Due to major personal and work changes in the past few months, I have not maintained the library as frequently as before. 

However, the library is still active (although it will be developed at a slower pace) and will be part, I hope, of a larger project in the coming months.

**Reply by trsh** [reply]
Is the larger project a secret? I am curious!
